pvHelpers/email_factory.py

```diff
@@ -101,16 +105,17 @@ def fromServerMessage(for_user_id, decrypted_msg, wrapped_key, key_version, mail
             ),
             "flags": decrypted_msg["flags"],
             "sender": {"user_id": decrypted_msg["private_metadata"]["sender"],
-                       "display_name": decrypted_msg["private_metadata"]["sender"]},
+                       "display_name": decrypted_msg["private_metadata"]["sender"],
+                       "external_email": decrypted_msg["private_metadata"].get("external_sender", None)},
```

Comment:
Does this add an entry in the dict if it's None? As in will it store into the relevant field some empty string or something?

Reply:
Stores it as None - I think originally this was done because of the checks later on in which checking for user["external_email"] errors out instead of returning None. I can comb through and change that though if we want to go that way.